core/dbt/parser/manifest.py

```diff
@@ -749,12 +743,42 @@ def _process_docs_for_exposure(
 # exposures: exposure descriptions
 def process_docs(manifest: Manifest, config: RuntimeConfig):
     for node in manifest.nodes.values():
-        ctx = generate_runtime_docs(
-            config,
-            node,
-            manifest,
-            config.project_name,
-        )
+        if isinstance(node, ParsedSchemaTestNode):
+            # Idealing we should be able to put the test kwargs in the context,
+            # but kwargs have already been processed at this point
+            # ex: 'model' --> "{{ ref('model') }}"
+            ctx = generate_runtime_docs(
+                config,
+                node,
+                manifest,
+                config.project_name,
+            )
+
+            model = []
+
+            if len(node.refs):
+                model = node.refs[0]
+            elif len(node.sources):
+                model = node.sources[0]
+
+            if len(model) == 1:
+                target_model_name = model[0]
+            elif len(model) == 2:
+                _, target_model_name = model
+            else:
+                raise dbt.exceptions.InternalException(
+                    f'Refs and sources should always be 1 or 2 arguments - got {len(model)}'
+                )
+
+            ctx['column_name'] = node.column_name
+            ctx['model'] = target_model_name
```

Comment:
blah hardcoded values set directly on the context, I tried to avoid this without success

Reply:
Ohh I see your comment above. Okay, that makes sense!